tpm2/tpm2.go

```diff
@@ -1459,6 +1459,71 @@ func Hash(rw io.ReadWriter, alg Algorithm, buf tpmutil.U16Bytes, hierarchy tpmut
 	return decodeHash(resp)
 }
 
+// HashSequenceStart starts a hash or an event sequence. If hashAlg is an
+// implemented hash, then a hash sequence is started. If hashAlg is
+// TPM_ALG_NULL, then an event sequence is started.
```

Comment:
Optional: Event sequences are started using the same command as Hash sequences, but they are terminated with another command. It looks like, since go-tpm already has unmarshalling code for TPML_DIGEST, adding EventSequenceComplete to this PR would be a fairly small update (~25 LOC). Would you like to add this? If not, would you mind adding a sentence here that says "by the way, event sequences are not currently supported by go-tpm" to save someone a bit of confusion later.

Reply:
I think adding `EventSequenceComplete` it's a better choice.
I'll give a look into this and get back to you in a moment.

Reply:
Personally, I de-dupe at three repetitions, and generally opine that earlier impacts readability.

Reply:
Just sent a commit adding `EventSequenceComplete`.
Please let me know what you think.

Reply:
This looks good, finishing up the review now.